tests/unit/Change.spec.ts

```diff
@@ -230,7 +234,24 @@ describe('Change component', () => {
     localVue.use(VueRouter)
     const router = mockRouter.mock()
     await router.push({ name: 'change' })
-    wrapper = shallowMount(Change, { localVue, store, router, vuetify })
+    wrapper = shallowMount(Change, { localVue,
+      store,
+      router,
+      vuetify,
+      data () {
```

Comment:
I can mock behavior for computed value changes by using the following code, this allowed me to use all of the pretesting/post testing methods instead of making new ones for a new computed value. I thought this was neat!